group_vars/all.yml

```diff
@@ -0,0 +1,35 @@
+---
+infrared_dir: "{{ ansible_user_dir }}/infrared"
+infrared_workspaces_dir: "~/.infrared/.workspaces"
+infrared_hosts_file: "{{ infrared_workspaces_dir }}/active/hosts"
+instackenv_file: "{{ ansible_user_dir }}/instackenv.json"
+cloud_name: cloud05
+lab_name: alias
+ansible_ssh_user: root
+ansible_ssh_pass: 100yard-
+ansible_ssh_key: "{{ ansible_user_dir }}/.ssh/id_rsa"
+osp_release: 13
+osp_puddle: passed_phase2
+controller_count: 3
+# No need to set compute_count. This will be set to all remaining nodes, which is calclulated in overcloud.yml
+#compute_count: 1
+alias:
+#lab specific vars
+  lab_url: "http://quads.alias.bos.scalelab.redhat.com"
+scale:
+# lab specific vars
+  lab_url: "http://quads.rdu2.scalelab.redhat.com"
+machine_types:
```

Comment:
We can get vendor name by reading sys entries
[root@f16-h10-000-1029p ~]# cat /sys/class/dmi/id/board_vendor
Supermicro
[root@e23-h33-740xd ~]# cat /sys/class/dmi/id/board_vendor
Dell Inc.
[vkommadi@localhost ~]$ cat /sys/class/dmi/id/board_vendor
LENOVO

But this change is more efficient which allows to find the vendor without login into the machine.

Reply:
I mean your change (i.e getting vendor by parsing url) is more efficient :)